server/web/src/components/dashboard/servers-panel.tsx

```
@@ -1,18 +1,181 @@
+          <button
+            onClick={onClose}
+            disabled={submitting}
+            className="text-slate-400 hover:text-slate-200 disabled:opacity-50"
+          >
+            <X className="w-5 h-5" />
+          </button>
+        </div>
+
+        <form onSubmit={handleSubmit} className="space-y-4">
+          <div>
+            <label className="block text-sm text-slate-400 mb-2">Server Name</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)